ui/components/Navigator.js

```diff
@@ -610,7 +610,20 @@ class Navigator extends React.Component {
           disableTouchListener={!drawerCollapsed}
         >
           <ListItemIcon className={classes.listIcon}>
-            <img src={icon} className={classes.icon} />
+            <img
+              src={icon}
+              className={classes.icon}
+              onMouseOver={(e) => {
+                e.target.style.transform = 'translate(-20%, -25%)';
+                e.target.style.top = '0';
+                e.target.style.right = '0';
+              }}
+              onMouseOut={(e) => {
+                e.target.style.transform = 'translate(0, 0)';
+                e.target.style.top = 'auto';
+                e.target.style.right = 'auto';
+              }}
+            />
```

Comment:
This is good, but can we use standardizing through the use of out themes? // @Modern-artist @ritiksaxena124

Reply:
I was trying to do that. But I don't know why it doesn't work. One potential reason can be -
  - The other icons of the navigator were coming natively but the meshmap icon was coming from the web. The other icons are JS versions of SVG but meshmap icon is pure SVG so the CSS wasn't working.